articles/aggregation-in-dbms/index.md

```diff
@@ -0,0 +1,78 @@
+Database management systems (DBMS) have replaced the traditional filing system by providing an easy, secure, efficient, and reliable way of storing, retrieving, accessing, and sharing data within databases. Some entities in a DBMS may have little value, which makes it difficult to use them for certain operations. In such situations, these entities can be combined with other entities to form a complex one that makes sense. This operation can be done through a process called aggregation. Aggregation in DBMS links trivial entities through relationships to ensure that the entire system functions well. 

```

Comment:
> Aggregation in DBMS links trivial entities through relationships to ensure that the entire system functions well.

You should introduce Entities and Relationships to the readers before starting to explain it. It would be easier for the reader to follow that way.

Reply:
Done.